addons/web/static/src/js/views/kanban/kanban_renderer.js

```diff
@@ -275,16 +287,76 @@ var KanbanRenderer = BasicRenderer.extend({
         this.$el.toggleClass('o_kanban_grouped', isGrouped);
         this.$el.toggleClass('o_kanban_ungrouped', !isGrouped);
         this.$el.empty();
-
         var fragment = document.createDocumentFragment();
         if (isGrouped) {
             this._renderGrouped(fragment);
         } else {
             this._renderUngrouped(fragment);
         }
         this.$el.append(fragment);
+        if (this.widgets && config.isMobile && isGrouped) {
+            core.bus.on("DOM_updated", this, function() {
+                this.enableSwipeOnRecordGroup();
+                this.moveToGroup(this.last_active_mobile_tab);
+            });
+            // Required to allow mobile kanban stage tabs to enable swipe and move to active tab
+            core.bus.trigger('DOM_updated');
+        }
         return this._super.apply(this, arguments).then(_.invoke.bind(_, oldWidgets, 'destroy'));
     },
+    enableSwipeOnRecordGroup: function() {
```

Comment:
1. method should be named with an underscore: _enableSwipeOnRecordGroup
2. it should have a docstring (and it should be marked @private
3. there should be a space between function and (
4. private methods should be ordered alphabetically. so this should not be below _renderView